modules/osdk-upgrading-v130-to-v180.adoc

```diff
@@ -0,0 +1,895 @@
+ helm-operator:
+-ifeq (,$(wildcard $(HELM_OPERATOR)))
+-ifeq (,$(shell which helm-operator 2>/dev/null))
++ifeq (, $(shell which helm-operator 2>/dev/null))
+ 	@{ \
+ 	set -e ;\
+-	mkdir -p $(dir $(HELM_OPERATOR)) ;\
+-	curl -sSLo $(HELM_OPERATOR) https://github.com/operator-framework/operator-sdk/releases/download/v1.3.0/helm-operator_$(OS)_$(ARCH) ;\
+-	chmod +x $(HELM_OPERATOR) ;\
++	mkdir -p bin ;\
++	curl -LO https://github.com/operator-framework/operator-sdk/releases/download/v1.8.0/helm-operator-v1.8.0-$(ARCHOPER)-$(OSOPER) ;\
++	mv helm-operator-v1.8.0-$(ARCHOPER)-$(OSOPER) ./bin/helm-operator ;\
++	chmod +x ./bin/helm-operator ;\
+ 	}
++HELM_OPERATOR=$(realpath ./bin/helm-operator)
+ else
+-HELM_OPERATOR = $(shell which helm-operator)
+-endif
++HELM_OPERATOR=$(shell which helm-operator)
+ endif
+----
+====
+
+.. Move the positional directory argument `.` in the `make` target for `docker-build`.
++
+The directory argument `.` in the `docker-build` target was moved to the last positional argument to align with `podman` CLI expectations, which makes substitution cleaner:
++
+.Old target
+[source,terminal]
+----
+docker-build:
+  docker build . -t ${IMG}
+----
++
+.New target
+[source,terminal]
+----
+docker-build:
+  docker build -t ${IMG} .
+----
++
+You can make this change by running the following command:
++
+[source,terminal]
+----
+$ sed -i 's/docker build . -t ${IMG}/docker build -t ${IMG} ./' $(git grep -l 'docker.*build \. ')
+----
+
+.. For Ansible- and Helm-based Operator projects, add a `help` target to the `Makefile`.
++
+Ansible- and Helm-based projects now provide `help` target in the `Makefile` by default, similar to a `--help` flag. You can manually add this target to your `Makefile` using the following lines:
++
+.`help` target
+[%collapsible]
+====
+[source,make]
+----
+##@ General
+
+# The help target prints out all targets with their descriptions organized
+# beneath their categories. The categories are represented by '##@' and the
+# target descriptions by '##'. The awk commands is responsible for reading the
+# entire set of makefiles included in this invocation, looking for lines of the
+# file as xyz: ## something, and then pretty-format the target and help. Then,
+# if there's a line with ##@ something, that gets pretty-printed as a category.
+# More info on the usage of ANSI control characters for terminal formatting:
+# https://en.wikipedia.org/wiki/ANSI_escape_code#SGR_parameters
+# More info on the awk command:
+# http://linuxcommand.org/lc3_adv_awk.php
+
+help: ## Display this help.
+	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_0-9-]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
+----
+====
+
+.. Add `opm` and `catalog-build` targets, which enable you to create your own catalogs for your Operator or add your Operator bundles to an existing catalog:
```

Comment:
I noticed a previous two sentence instruction text.  Just sharing a thought to consider splitting at the comma and starting with "These targets enable you..."

I have no sugg for replacing "enable" if you have personification concerns.

Reply:
I'll switch to "You can use these targets to..."